server/module/services/router/inference_router.py

```diff
@@ -72,9 +72,17 @@ async def logging_route_handler(request: Request) -> Response:
             finally:
                 if request.state._state.get("api_key_data_tracking"):
                     req_json: Dict[str, Any] = json.loads(req_body)
-                    enable_tracking = req_json.get(
-                        "controlConfig", {"dataTracking": True}
-                    )["dataTracking"]
+                    # enable_tracking = req_json.get(
+                    #     "controlConfig", {"dataTracking": True}
+                    # )["dataTracking"]
+
+                    controlConfig = req_json.get(
+                        "controlConfig", {}
+                    )
+                    if "dataTracking" not in controlConfig:
+                        enable_tracking = True
```

Comment:
🥲

Reply:
Why not just do 
`enable_tracking = True if "dataTracking" not in controlConfig else False`

Reply:
Or you know what, this works fine. I am merging.